Misc/NEWS.d/next/Library/2018-10-13-18-16-20.bpo-31522.rWBb43.rst

```diff
@@ -0,0 +1 @@
+In `mailbox._mboxMMDF.get_string`, pass *from_* parameter to ``get_bytes``.
```

Comment:
This describes what was done, but the changelog should in general describe the externally visible affects of the change.  In this case, I would write "The mailbox mbox get_string function `from_` parameter can now successfully be set to a non-default value".